modules/extensions-support/src/main/java/org/mule/runtime/module/extension/internal/resources/documentation/ExtensionDocumentationResourceGenerator.java

```diff
@@ -52,7 +51,7 @@ public Optional<GeneratedResource> generateResource(ExtensionModel extensionMode
                                                            walker.getOperations(),
                                                            walker.getSources(),
                                                            getTypesDocumentation(extensionModel)));
-    return Optional.of(new GeneratedResource(SERIALIZER.getFileName(extensionModel.getName()), documenter.getBytes()));
+    return of(new GeneratedResource(true, "META-INF/" + SERIALIZER.getFileName(extensionModel.getName()), documenter.getBytes()));
```

Comment:
May this fixed "/" be an issue in different OSs?

Reply:
where this is used, a `resourceName` is expected (AnnotationProcessorResourceGenerator#L57) rather than a filePath, so forward slash is the only valid thing.